app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/SigningUp.scala.html

```diff
@@ -0,0 +1,40 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.config.AppConfig
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.views.html.templates.PrincipalMainTemplate
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.views.html.helpers.ContinueButton
+
+@this(
+mainTemplate: PrincipalMainTemplate,
+form: FormWithCSRF,
+continueButton: ContinueButton
+)(
+implicit appConfig: AppConfig
+)
+
+
+@(postAction:Call)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(title = messages("individual.signing-up.heading"), backLink = None) {
+
+<h1 class="govuk-heading-l">@messages("individual.signing-up.heading")</h1>
+
+@form(action = postAction) {
+@continueButton(alternateText = Some(messages("base.accept-and-continue")))
```

Comment:
The prototype has a `Continue` button:
```suggestion
@continueButton()
```

Reply:
Updated a recommended